### DISCUSSION TITLE
I'm not able to use github pages.

### DISCUSSION THREAD

**Discussion by crmdevtool** [community/community#35861] [Other Feature Feedback, Questions, & Ideas]
:(

**Comment by DSDmark** [comment]
Hey,
kindly add some more detail related to your problem for better understanding 

if you use any JS framework to publish your code to GitHub, then you need to first build the application for production usage

> [Here is Artical](https://javascript.plainenglish.io/deploying-any-app-to-github-pages-1e8e946bf890)
